modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -7907,6 +7927,11 @@ type Query {
   """
   proposalAttachmentTypeMeta: [ProposalAttachmentTypeMeta!]!
 
+  """
+  Metadata for `enum ProposalStatus
+  """
+  proposalStatusMeta: [ProposalStatusMeta!]!
```

Comment:
I assume there will be quite a few of these before we're done.  Maybe they should be separated into a `EnumMetaType` or something in order to keep the individual entries out of the top level?

Reply:
Seems like a reasonable idea. The rest api that provides the enum metadata would just need to be changed to use the new path. Explore (and probably the other UIs) don't care about it directly in the GraphQL API because they get the metadata through the rest api to create the dynamic enums.